aws-xray-recorder-sdk-aws-sdk/src/main/resources/com/amazonaws/xray/handlers/DefaultOperationParameterWhitelist.json

```diff
@@ -325,6 +325,409 @@
           ]
         }
       }
+    },
+    "Amazon S3": {
+      "operations": {
+        "CopyObject": {
+          "request_parameters": [
+            "SourceBucketName",
+            "SourceKey",
+            "DestinationBucketName",
+            "DestinationKey"
+          ]
+        },
+        "CopyPart": {
+          "request_parameters": [
+            "SourceBucketName",
+            "SourceKey",
+            "DestinationBucketName",
+            "DestinationKey"
+          ]
+        },
+        "GetObject": {
+          "request_parameters": [
+            "Key",
+            "VersionId",
+            "BucketName"
+          ]
+        },
+        "PutObject": {
+          "request_parameters": [
+            "Key",
+            "BucketName"
+          ]
+        },
+        "GetObjectAcl": {
+          "request_parameters": [
+            "Key",
+            "VersionId",
+            "BucketName"
+          ]
+        },
+        "CreateBucket": {
+          "request_parameters": [
+            "BucketName"
+          ]
+        },
+        "ListObjectsV2": {
+          "request_parameters": [
+            "Prefix",
+            "BucketName"
+          ]
+        },
+        "ListObjects": {
+          "request_parameters": [
+            "Prefix",
+            "BucketName"
+          ]
+        },
+        "GetObjectTagging": {
+          "request_parameters": [
+            "Key",
+            "VersionId",
+            "BucketName"
+          ]
+        },
+        "SetObjectTagging": {
+          "request_parameters": [
+            "Key",
+            "VersionId",
+            "BucketName"
+          ]
+        },
+        "ListVersions": {
+          "request_parameters": [
+            "Prefix",
+            "BucketName"
+          ]
+        },
+        "SetObjectAcl": {
+          "request_parameters": [
+            "Key",
+            "VersionId",
+            "BucketName"
+          ]
+        },
+        "GetBucketAcl": {
+          "request_parameters": [
+            "BucketName"
+          ]
+        },
+        "SetBucketAcl": {
+          "request_parameters": [
+            "BucketName"
+          ]
+        },
+        "HeadBucket": {
+          "request_parameters": [
+            "BucketName"
+          ]
+        },
+        "UploadPart": {
+          "request_parameters": [
+            "Key",
+            "BucketName"
+          ]
+        },
+        "DeleteObject": {
+          "request_parameters": [
+            "Key",
+            "BucketName"
+          ]
+        },
+        "DeleteBucket": {
+          "request_parameters": [
+            "BucketName"
+          ]
+        },
+        "DeleteObjects": {
```

Comment:
It would be interesting to add

```json
          "response_descriptors": {
            "DeletedObjects": {
              "list": true,
              "get_count": true,
              "rename_to": "deleted_object_count"
            }
          }
```
here.